packages/wallet-ui/src/utils/utils.ts

```diff
@@ -125,3 +125,15 @@ export const fetchWithTimeout = async (resource: string, options = { timeout: TI
   clearTimeout(id);
   return response;
 };
+
+export const shortenDomain = (domain: string, maxLength = 18) => {
+  if (!domain) return '';
+  const ellipsis = '...';
+
+  if (domain.length <= maxLength) {
```

Comment:
does the domain always in eng? if not we may consider to verify the length by [...domain].length

Reply:
For now yes, but it could change in the future so I updated this part